docs/translations/api-docs/dialog/dialog.json

```diff
@@ -3,6 +3,7 @@
   "propDescriptions": {
     "aria-describedby": { "description": "The id(s) of the element(s) that describe the dialog." },
     "aria-labelledby": { "description": "The id(s) of the element(s) that label the dialog." },
+    "aria-modal": { "description": "Indicates whether an element is modal when displayed." },
```

Comment:
This isn't exactly true. The `aria-modal` attribute does not control the modality but just informs assistive technologies that the element is modal.

Reply:
Don't you think it's worth explicitly defining it in our props? Its behavior is different, as all other `aria-` props are spread on the root, while this one is placed on the inner Paper component.

Reply:
Yeah ok, fair enough, I will add it.